examples/src/main/java/com/mapbox/navigation/examples/core/replay/HistoryRecorder.kt

```diff
@@ -43,7 +43,6 @@ class HistoryRecorder(
      */
     fun startRecording() {
         startedAt = Date()
-        navigation.toggleHistory(true)
     }
 
```

Comment:
This needs a whole overhaul as History recording APIs changed drastically cc @kmadsen